shared/constants/fs.js

```diff
@@ -75,6 +75,7 @@ export const makeState: I.RecordFactory<Types._State> = I.Record({
   kbfsInstalling: false,
   fuseInstalling: false,
   kextPermissionError: false,
+  showBanner: false,
```

Comment:
It'd be nice if all of FUSE related states can go into a separate `fuse` field which is its own type.

Reply:
I'll make a separate refactor PR for that.